.depcheckrc.json

```diff
@@ -0,0 +1,14 @@
+{
+    "ignore-patterns": ["libDev"],
+    "ignores": [
+        "babel*",
+        "karma*",
+        "@babel/*",
+        "@playwright/*",
+        "webpack*",
+        "worker-loader",
+        "node-libs-browser",
+        "rimraf"
```

Comment:
i would add `tslib` as well - it's marked as "peerDependency" in connect and other libs because it was constantly removed as unused

Reply:
added in commit: [70b9074](https://github.com/trezor/trezor-suite/pull/12914/commits/70b90749b22d908b7f8860f0972b528bf2dbfdf1)